feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -154,19 +154,16 @@ func bgpCreateNbr(localAs, peerAs uint32, policy string) *telemetry.NetworkInsta
 	bgp := ni_proto.GetOrCreateBgp()
 	ni_proto.Name = ygot.String("BGP")
 	ni_proto.Identifier = telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP
-	ni_proto.Enabled = ygot.Bool(true)
 	global := bgp.GetOrCreateGlobal()
 	global.RouterId = ygot.String(dutDst.IPv4)
 	global.As = ygot.Uint32(localAs)
 	global.GetOrCreateAfiSafi(telemetry.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
```

Comment:
Do we still need this one?

Reply:
No, this commit actually removes `ni_proto.Enabled = ygot.Bool(true)`

Reply:
My bad, i linked the wrong line.

I mean do we still need line 161:
global.GetOrCreateAfiSafi(telemetry.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)

Reply:
This line is actually required